### ISSUE TITLE
[CNVM] CloudFormation instance doesn't have enough space

### ISSUE THREAD

**Issue by amirbenun** [elastic/cloudbeat#1098]
### Describe the bug
Cloudbeat prints `no space left on device` error and EC2 shuts down later
```
[elastic_agent.cloudbeat][error] VulnerabilityScanner.scan.ScanVM error: scan error: scan failed: failed analysis: inspection error: post analysis error: post analysis error: Java DB update failed: Java DB update error: DB download error: oci download error: download error: failed to download: write java-db/trivy-java.db: no space left on device
```

### Preconditions
8.9 ESS stacks

**Comment by amirbenun** [comment]
In order to verify:
1. Create a CNVM integration
2. Deploy using CloudFormation
3. On AWS console, check the Storage tab of the EC2 instance created as part of the CloudFormation stack
4. Make sure that:
  a. It has a single volume
  b. The volume size is 32Gb
  c. The volume is the root device

![image](https://github.com/elastic/cloudbeat/assets/34831306/9ce0c69a-359b-4711-a556-eefe533af59f)

**Comment by tehilashn** [comment]
After syncing with @oren-zohar - let's backport this to 8.8, if there will be a 8.8.3 release then we'll have this fix

**Comment by amirbenun** [comment]
Sounds good @tehilashn.
I opened a PR with the backport to 8.8
https://github.com/elastic/cloudbeat/pull/1115

**Comment by amirbenun** [comment]
Actually we don't need a release for this change, we can just publish the new integration version.
Once we will merge this https://github.com/elastic/integrations/pull/6830 it will trigger our new version build and it will be publicly available for all 8.8 environments.

**Comment by tehilashn** [comment]
Adding Verified as it was tested
